astropy/cosmology/flrw/w0wzcdm.py

```diff
@@ -273,7 +273,7 @@ class Flatw0wzCDM(FlatFLRWMixin, w0wzCDM):
     The comoving distance in Mpc at redshift z:
 
     >>> cosmo.comoving_distance(0.5)
-    <Quantity 1982.66012926 Mpc>
+    <Quantity 1849.74726272 Mpc>
```

Comment:
Is there a way to check those numerical values with another package/software ?

Reply:
I can test this with Mathematica.
I'll add some tests.

Reply:
Confirmed:
<img width="722" alt="CleanShot 2023-07-22 at 17 05 32@2x" src="https://github.com/astropy/astropy/assets/8949649/2d17e67a-2e8f-4eb6-b3b8-7cf15d79dbfc">

I'll add a test.

Reply:
Done!

Reply:
Same example is now a test, with the Mathematica code if this ever needs to be rerun.

Reply:
Ping @saimn.

Reply:
@saimn, I implemented a test.

Reply:
@pllim, I've checked the calculation (see screenshot in this thread) and implemented a trivial test. I think this is an uncontroversial merge...

Reply:
Can you please rebase then, the CI should be green by now.

Reply:
Done.